example/test_problem/Hydro/CCSN/Input__TestProb.CoreCollapse

```diff
@@ -20,15 +20,15 @@ CCSN_Eint_Mode   1                                 # Mode of obtaining internal
                                                    # ( 1=Temp Mode: Eint(dens, temp, [Ye])
                                                    #   2=Pres Mode: Eint(dens, pres, [Ye]) )
 
-CCSN_MaxRefine_FLAG1   1                           # enable limiting maximum refinement level 1 [0]
-CCSN_MaxRefine_FLAG2   1                           # enable limiting maximum refinement level 2 [0] ( CCSN_MaxRefine_FLAG1 should be turn on first)
-CCSN_MaxRefine_LV1     7                           # reduced maximum refinement level 1 to this value [MAX_LEVEL-2]
-CCSN_MaxRefine_LV2     7                           # reduced maximum refinement level 2 to this value [MAX_LEVEL-1]
-CCSN_MaxRefine_Dens1   1.0e11                      # central density threshold that reduces the maximum refinement level 1 [1.0e11]
-CCSN_MaxRefine_Dens2   1.0e12                      # central density threshold that reduces the maximum refinement level 2 [1.0e12]
-CCSN_MaxRefine_RadFac  0.15                        # factor that determines the maximum refinement level based on distance from the box center  [0.15]
-CCSN_CC_CentralDensFac 1.0e13                      # factor that reduces the dt constrained by the central density (in cgs) during the core collapse  [1.0e13]
-CCSN_CC_Red_DT         1.0e-5                      # Reduced time step (in s) when the central density exceeds CCSN_CC_CentralDensFac before bounce [1e-5]
-CCSN_LB_TimeFac        0.10                        # factor that scales the dt constrained by lightbulb scheme  [0.1]
+CCSN_CC_MaxRefine_Flag1   0                        # enable limiting maximum refinement level 1 [0]
+CCSN_CC_MaxRefine_Flag2   0                        # enable limiting maximum refinement level 2 [0]
```

Comment:
Remember that the template aims for reproducing the core collapse test without modifying the parameters. Now since you change the default value to 0, I am wondering whether it is able to reproduce the test without these two refinement criteria?

Reply:
I changed the default values to `1`